### PR TITLE
OC-380 Makes the NotificationService suitable for multiple endpoints

### PR DESCRIPTION
**Pull request by bvdzwet**
Adds a WebServiceConfigurationDataBuilder to assist in creating the
WebServiceConfigurationData entities holding web service notification
endpoint configuration that was previously available to the application
through properties files.
Makes sure the necessary web service configuration records are inserted
as default data during Cucumber database preparation.

### CODE REVIEW

File: cucumber-tests-platform-microgrids/src/test/java/org/opensmartgridplatform/cucumber/platform/microgrids/glue/steps/database/ws/WebServiceConfigurationDataBuilder.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.cucumber.platform.microgrids.glue.steps.database.ws;
+
+import org.opensmartgridplatform.adapter.ws.domain.entities.WebServiceConfigurationData;
+import org.opensmartgridplatform.adapter.ws.domain.entities.NotificationWebServiceLookupKey;
+
+public class WebServiceConfigurationDataBuilder {
```
Comment (smvdheijden) [inline cucumber-tests-platform-microgrids/src/test/java/org/opensmartgridplatform/cucumber/platform/microgrids/glue/steps/database/ws/WebServiceConfigurationDataBuilder.java]: Rename to NotificationWebServiceConfigurationBuilder?
When NotificationWebServiceConfiguration management functions are added, this class would probably be moved to Platform?

**Comment by bvdzwet** [comment]
retest this please

**Comment by bvdzwet** [comment]
retest this please

**Comment by smvdheijden** [comment]
Retest this please

**Pull request merged**